### ISSUE TITLE
Passing objects without {} sometimes ambiguous

### ISSUE THREAD

**Issue by familyfriendlymikey** [imba/imba#876]
### Description

Works as intended:
```
cb! if await model.update { bool: yes }
```

Does not work as intended:
```
cb! if await model.update
	bool: yes
```

**Comment by haikyuu** [comment]
I find the second example very confusing. I understand if the parser feels the same 😅

**Comment by familyfriendlymikey** [comment]
I just think if it's ambiguous like this then the compiler should throw or something.

**Comment by haikyuu** [comment]
Yeah, I've also faced this quite a bit recently. The app isn't working, and I receive no error. I usually work around it by manually compiling `imbac suspect.imba` which is obviously not a good experience. Sometimes it takes time to know what's going on!

**Comment by madmaniak** [comment]
The second one looks like a hash in an if statement - so what should compiler throw in this situation?
In some languages if you wanted to stick to the previous line you would use `\` before new line - and this way it would be interpreted as you intended.

**Comment by madmaniak** [comment]
Didn't notice the `cb()` method - in that case I have no idea how to interpret it.
